grakn-test/test-snb/src/main/java/ai/grakn/GraknDb.java

```diff
@@ -52,11 +54,11 @@ protected void onInit(Map<String, String> properties, LoggingService loggingServ
 
         // TODO: disabling a lot because they are slow
         registerOperationHandler(LdbcShortQuery1PersonProfile.class, GraknShortQueryHandlers.LdbcShortQuery1PersonProfileHandler.class);
-//        registerOperationHandler(LdbcShortQuery2PersonPosts.class, GraknShortQueryHandlers.LdbcShortQuery2PersonPostsHandler.class);
-//        registerOperationHandler(LdbcShortQuery3PersonFriends.class, GraknShortQueryHandlers.LdbcShortQuery3PersonFriendsHandler.class);
+        registerOperationHandler(LdbcShortQuery2PersonPosts.class, GraknShortQueryHandlers.LdbcShortQuery2PersonPostsHandler.class);
```

Comment:
You probably know this but this has no affect because the execution of this in the `JenkinsFile` is still commented out.